src/parseFunctions.js

```diff
@@ -0,0 +1,21 @@
+import {keyTypes} from './constants.js'
+
+export const parseQueryToJson=(queryString)=>{
+    const values=decodeURIComponent(queryString).split('CMCD=')[1].split('&')[0].split(',');
+    const obj={};
+    for(const value of values){
+        const [key,val]=value.split('=');
+        if(keyTypes[key] ==='Integer'){
+            obj[key]=parseInt(val);
+        }else if(keyTypes[key]==='Boolean'){
+            if(val==='false'){
+                obj[key]=false;
+            }else{
+                obj[key]=true;
+            }
```

Comment:
To avoid having multiple nested `if` statements, maybe convert this to a one-liner like this:
```
val === 'false' ? obj[key] = false : obj[key] = true;
```

Also, i wonder if the input value is something different than `'false'`, it'll always default to a `true` value. For a first approach maybe this is ok, but might need to check if we can improve this logic in the future.